integration-cli/docker_cli_daemon_test.go

```diff
@@ -1527,3 +1527,26 @@ func teardownV6() error {
 	}
 	return nil
 }
+
+func (s *DockerDaemonSuite) TestDaemonRestartContainerWithRestartPolicyAlwaysManuallyStoppedRestart(c *check.C) {
```

Comment:
I really don't get this name. I'm pretty sure even `TDRCRAPMS` would be more readable for me :)